src/jsonapi-vuex.js

```diff
@@ -359,6 +359,37 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const addJvHelpers = (obj) => {
+  // Add Utility functions to _jv child object
+  Object.assign(obj[jvtag], {
+    isRel(name) {
+      return obj[jvtag]['relationships'].hasOwnProperty(name)
```

Comment:
worth taking out obj[jvtag]['relationships'].hasOwnProperty(name) to some local isRel or something, or reusing isRel in other calls - it's used 3 times in this snippet

Reply:
Have set other the calls to reuse `isRel`